lib/os/p4wq.c

```diff
@@ -143,7 +143,11 @@ static int static_init(const struct device *dev)
  * so they can initialize in parallel instead of serially on the main
  * CPU.
  */
+#ifdef CONFIG_SMP
 SYS_INIT(static_init, SMP, 99);
+#else
+SYS_INIT(static_init, APPLICATION, 99);
+#endif
```

Comment:
Just pick one.  There's no reason not to use APPLICATION all the time, this doesn't have any significant dependancies we need to wait for.

Though... we should probably be defining the SMP initlevel anyway just for compatibility, to avoid code like this...